python/modules/Stop0l_trigger.py

```diff
@@ -235,22 +233,36 @@ def analyze(self, event):
 			zmumu_mid.append(zmumu_cand)
 	n_zmumu = len(zmumu_mid)
 
-	photon_loose = []
-	for photon in photons:
-		if (self.SelPhotons(photon)):
-			photon_loose.append(photon)
-	n_photon = len(photon_loose)
+        photon_loose = []
+        photon_mid = []
+        for photon in photons:
+                if (abs(photon.eta) < 1.442 or (1.566 < abs(photon.eta) and abs(photon.eta) < 2.5)):
```

Comment:
Please use the same photon eta cut. See

https://github.com/susy2015/NanoSUSY-tools/blob/master/python/modules/Stop0lObjectsProducer.py#L143-L144